Dev-Corner/slides/tool_integration.html

```diff
@@ -442,7 +444,7 @@
 graphlan.py
 --format '$format'
 &&
-echo "Yeeha it worked"
+echo "Yeah it worked!"
```

Comment:
Look at the presidents of the world ... I guess we are. Sad story.
Thanks Nicola!